pkg/operator/ceph/csi/util.go

```diff
@@ -61,26 +62,40 @@ func templateToDaemonSet(name, templatePath string, p templateParam) (*apps.Daem
 	var ds apps.DaemonSet
 	t, err := loadTemplate(name, templatePath, p)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed loading daemonset template. %+v", err)
```

Comment:
not sure whats better, but mentioning it here :

failed loading or failed to load ? :)

Reply:
updated to `failed to load`

Reply:
@Madhu-1 looks like there are more instances in the PR on `failed loading` , so whatever we pick we have to change in all the places.

Reply:
no @humblec  can you please recheck.